lib/rubocop/cop/style/redundant_ternary.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+# TODO: when finished, run `rake generate_cops_documentation` to update the docs
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for redundant uses of true/false in ternary conditions
+      #
+      # @example
+      #   # bad
+      #   x == y ? true : false
+      #
+      #   # good
+      #   x == y
+      #
+      #   # bad
+      #   x == y ? false : true
+      #
+      #   # good
+      #   x != y
+      class RedundantTernary < Cop
+        include ConfigurableEnforcedStyle
+
+        OPS = %w[== === != < > <= >= <=>].freeze
```

Comment:
Ah nice, I took this from the Lint/UselessComparison cop but good to know there's a constant, I can only see it used in Style/YodaCondition